pkg/services/ngalert/notifier/channels/opsgenie.go

```diff
@@ -179,13 +187,17 @@ func (on *OpsgenieNotifier) buildOpsgenieMessage(ctx context.Context, alerts mod
 	var tmplErr error
 	tmpl, data := TmplText(ctx, on.tmpl, as, on.log, &tmplErr)
 
-	title := tmpl(DefaultMessageTitleEmbed)
-	description := fmt.Sprintf(
-		"%s\n%s\n\n%s",
-		tmpl(DefaultMessageTitleEmbed),
-		ruleURL,
-		tmpl(`{{ template "default.message" . }}`),
-	)
+	title := tmpl(on.Message)
+
+	description := tmpl(on.Description)
+	if description == "" {
```

Comment:
I wonder what happens if the user puts a space here, i.e. " ". I wonder if we should trim and then check